test/unit/spec/Registration_spec.js

```diff
@@ -103,6 +103,40 @@ function (Q, _, $, OktaAuth, Util, Expect, Beacon, RegistrationForm, Router, $sa
       });
     });
 
+    Expect.describe('password complexity', function () {
+      itp('shows password complexities unsatisfied', function () {
+        return setup().then(function (test) {
+          expect(test.form.hasPasswordComplexityUnsatisfied('minLength')).toBe(true);
+          expect(test.form.hasPasswordComplexityUnsatisfied('minLowerCase')).toBe(true);
+          expect(test.form.hasPasswordComplexityUnsatisfied('minUpperCase')).toBe(true);
+          expect(test.form.hasPasswordComplexityUnsatisfied('minNumber')).toBe(true);
+          expect(test.form.hasPasswordComplexityUnsatisfied('excludeUsername')).toBe(true);
```

Comment:
Maybe also add an ``expect`` that the ``hasPasswordComplexitySatisfied`` and ``hasPasswordComplexityError`` are false for these fields.